core/services/gateway/handlers/webapicapabilities/webapi.go

```diff
@@ -15,3 +19,51 @@ type TargetResponsePayload struct {
 	Headers      map[string]string `json:"headers,omitempty"`       // HTTP headers
 	Body         []byte            `json:"body,omitempty"`          // HTTP response body
 }
+
+// https://gateway-us-1.chain.link/web-trigger
+//   {
+//     jsonrpc: "2.0",
+//     id: "...",
+//     method: "web-trigger",
+//     params: {
+//       signature: "...",
+//       body: {
+//         don_id: "workflow_123",
+//         payload: {
+//           trigger_id: "web-trigger@1.0.0",
+//           trigger_event_id: "action_1234567890",
+//           timestamp: 1234567890,
+//           topics: ["daily_price_update"],
+//           params: {
+//             bid: "101",
+//             ask: "102"
+//           }
+//         }
+//       }
+//     }
+//   }
+// from Web API Trigger Doc
+// trigger_id          - ID of the trigger corresponding to the capability ID
+// trigger_event_id    - uniquely identifies generated event (scoped to trigger_id and sender)
+// timestamp           - timestamp of the event (unix time), needs to be within certain freshness to be processed
+// topics            - [OPTIONAL] list of topics (strings) to be started by this event (affects all topics if empty)
```

Comment:
Update for https://github.com/smartcontractkit/chainlink/pull/14580/files#diff-d12119c838c10d3ed4d6d8bd6fe57b297a9c4044630306265e2870da8a37cbc9R95 , this is no longer optional

Reply:
I say lower down in an amendments section that it must be specified and a single topic.  But I can see how that's missed so I edited the field descriptions as you suggest.